app/v1/support/DesResponseMappingSupport.scala

```diff
@@ -33,6 +35,28 @@ trait DesResponseMappingSupport {
     }
   }
 
+  final def validateCodingOutResponse[T](desResponseWrapper: ResponseWrapper[T], taxYear: String)
```

Comment:
It's probably better to use strict typing here since this function should only be used for RetrieveCodingOutResponse objects

Reply:
i.e - ResponseWrapper[RetrieveCodingOutResponse]